includes/templates/responsive_classic/templates/tpl_shopping_cart_default.php

```diff
@@ -81,9 +81,9 @@
   } else {
     echo $product['quantityField'];
   }
-?><br>
-        <span class="alert bold"><?php echo $product['flagStockCheck'];?></span><br>
-        <br><?php echo $product['showMinUnits']; ?>
+  echo !empty($product['flagStockCheck']) ? '<span class="alert bold">' . $product['flagStockCheck'] . '</span><br>' : '';
+  echo !empty($product['showMinUnits']) ? $product['showMinUnits'] . '<br>' : '';
+  ?>
```

Comment:
Why remove the `<br>` tags? These spans aren't block-level, right, so they'll just sit beside each other, instead of giving the visitor visual cues of their difference?

Reply:
Fair. I was focusing on the first `<br>`, which has been removed, and that the double-`<br>` between lines has been changed to a single.